simple_graph.py

```diff
@@ -80,13 +87,17 @@ def _depth_first_helper(self, node, visited):
         return visited
 
     def depth_first_traversal(self, node):
-        """Perform a depth-first traversal of the graph. Return a list of
-         nodes in the order visited."""
+        """
+        Perform a depth-first traversal of the graph. Return a list of
+        nodes in the order visited.
+        """
         return self._depth_first_helper(node, [])
 
     def breadth_first_traversal(self, node):
-        """Perform a breadth-first traversal of the graph. Return a list of
-         nodes in the order visited."""
+        """
+        Perform a breadth-first traversal of the graph. Return a list of
+        nodes in the order visited.
+        """
```

Comment:
The second sentence in this docstring should be the first.  It's much more useful than the first.